src/main/java/com/shaneschulte/mc/clanarena/commands/CreateCmd.java

```diff
@@ -0,0 +1,379 @@
+package com.shaneschulte.mc.clanarena.commands;
+
+import com.shaneschulte.mc.clanarena.utils.CmdProperties;
+import com.shaneschulte.mc.clanarena.utils.MsgUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.boss.BarColor;
+import org.bukkit.boss.BarStyle;
+import org.bukkit.boss.BossBar;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.Action;
+import org.bukkit.event.entity.PlayerDeathEvent;
+import org.bukkit.event.player.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class CreateCmd implements CmdProperties, Listener {
+
+    private static HashMap<String, BossBar> currentPlayerBossBars = new HashMap<>();
+    private static HashMap<String, String> currentArenaNameByPlayer = new HashMap<>();
+    private static int minSpawns = 6;
+
+    // Stage 01: selecting region
+    private static ArrayList<String> playersInTheMiddleOfDesignatingThe2PointsNeededToCreateAnArenaRegion = new ArrayList<>();
```

Comment:
this symbol name is terrifying

Reply:
Create an enum to track which stage of the creation process the player is in instead (call it ArenaSetupState or something like that)